docs/tasks.md

```diff
@@ -555,13 +553,6 @@ metadata:
   name: task-with-parameters
 spec:
   params:
-    - name: gitrepo
```

Comment:
Curious what's the reason to remove this? I think it's useful to have example usage in the docs as well as in the examples folder

Reply:
sg! added them back. Thanks